### ISSUE TITLE
[MCU] Implement Real Bus in CAN Driver

### ISSUE THREAD

**Issue by AndreasFreund** [amosproj/amos2024ss07-updating-flash-boot-loader#31]
## User story
1. As a developer
2. I want to send CAN messages to other nodes on the network
3. So that I can communicate with the vector adapter

## Acceptance criteria
* Demonstrate MCU <-> Vector Communication (CANoe or demo app from Michael)
* Bitrate adjustable somehow (init function parameters or during runtime)

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by sebas-rodri** [comment]
@ibexq will review my code
